scripts/save.rb

```diff
@@ -0,0 +1,31 @@
+require 'csv'
+
+desc "Import houses from csv file"
```

Comment:
Good job taking a swing at importing the CSV. you would use this syntax if you were creating an import task rake task though